bin/genesis

```diff
@@ -665,6 +665,9 @@ $GLOBAL_USAGE
   -L, --link-dev-kit  Instead of using a kit or initializing an empty dev
                       directory, this will link the specified directory to the
                       dev directory.
+      --vault         Name of a 'safe' target (a Vault) to store newly generated
```

Comment:
Can we write this with a leading backquote?  as in:

```
Name of a `safe' target ...
```

Reply:
Really....?

Reply:
It's called a "Grave Accent``